modules/ossm-kiali-ossmc-plugin-user-guide.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+//* *service_mesh/v2x/ossm-kiali-ossmc-plugin.adoc
+
+
+:_mod-docs-content-type: CONCEPT
+[id="ossm-kiali-ossmc-plugin-user-guide_{context}"]
+= About the OpenShift Service Mesh Console plugin
+//In the title include nouns or noun phrases that are used in the body text.
+//Do not start the title of concept modules with a verb.
+
+The {SMProductName} Console (OSSMC) plugin is an extension to the {product-title} web console that provides visibility into your Service Mesh. 
+
+[WARNING]
+====
+The {SMPRoductName} Console (OSSMC) plugin only supports a single Kiali instance. Whether that Kiali instance is configured to access only a subset of OpenShift projects or has access cluster-wide to all projects does not matter, however, only a single Kiali instance can be accessed.
```

Comment:
```suggestion
The {SMPRoductName} Console (OSSMC) plugin only supports a single Kiali instance. Whether that Kiali instance is configured to access only a subset of OpenShift projects or has access cluster-wide to all projects does not matter. However, only a single Kiali instance can be accessed.
```
I think punctuation is needed to avoid a run-on sentence.